frontend/src/features/admin-form/settings/mutations.ts

```diff
@@ -278,6 +280,19 @@ export const useMutateFormSettings = () => {
     },
   )
 
+  const mutateFormEndPage = useMutation(
```

Comment:
similarly let's move this out of the `settings` folder as it's going to be shared across Create and Settings